html/statistics.html

```diff
@@ -0,0 +1,23 @@
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
+</head>
+
+<div id="main">
+  <body>
+    <div class="division dark">
+      <div class="box dark"></div>
+      <div class="box light" style="grid-column: 1 / 13; padding: 10px;">
+        <h2 class="box_value" style="font-size: xxx-large;">Win/Loose</h2>
```

Comment:
`xxx-large`is not fully supported (occurs 2 times)
https://developer.mozilla.org/de/docs/Web/CSS/font-size
https://www.chromestatus.com/feature/5675317559558144